.github/workflows/build-images-action.yml

```diff
@@ -1,56 +1,27 @@
 name: build-images-action
 
 on:
-  push:
-    branches:
-    - 'main'
-    - 'release-*'
-    tags:
-    - 'v*'
-
-permissions: {}
+  pull_request
+  # push:
```

Comment:
Let's wait with removing these until secrets are in place and we can see the push working too, otherwise LGTM.